src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  sortedArray: Todo[];
+  loadingTodo: Todo | null;
+  errorFunction: (el: string) => void;
+  deletingFunction: (el: boolean) => void;
+  deletingListId: number[];
+  todosFunction: (
+    todosOrCallback: Todo[] | ((todos: Todo[]) => Todo[]),
+  ) => void;
+  todos: Todo[];
+  focusInput: () => void;
+  activateTodosId: number[];
+};
+
+export const TodoList: React.FC<Props> = ({
+  sortedArray,
+  loadingTodo,
+  errorFunction,
+  deletingFunction,
+  deletingListId,
+  todosFunction,
+  todos,
+  focusInput,
+  activateTodosId,
+}) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {sortedArray.map((todo: Todo) => (
+        <TodoItem
+          key={todo.id}
+          todo={todo}
+          todosFunction={todosFunction}
```

Comment:
how can some other developer understand what todosFunction will do, better to rename it